packages/truffle/test/scenarios/commands/obtain.js

```diff
@@ -0,0 +1,62 @@
+const MemoryLogger = require("../MemoryLogger");
+const CommandRunner = require("../commandRunner");
+const Config = require("@truffle/config");
+const fse = require("fs-extra");
+const path = require("path");
+const assert = require("assert");
+const sandbox = require("../sandbox");
+
+let logger, config, project, expectedPath;
+
+describe("truffle obtain", function () {
+  project = path.join(__dirname, "../../sources/develop");
```

Comment:
Obtain should use its own test fixture, I think.

Reply:
yep, I forgot to add one for it